torch/distributed/nn/api/remote_module.py

```diff
@@ -210,109 +211,109 @@ def remote_parameters(self, recurse: bool = True) -> List[rpc.RRef[Parameter]]:
 
     def register_buffer(
         self, name: str, tensor: Optional[Tensor], persistent: bool = True
-    ) -> None:
+    ) -> None:  # type: ignore[return]
         _raise_not_supported(self.register_buffer.__name__)
 
-    def register_parameter(self, name: str, param: Optional[Parameter]) -> None:
+    def register_parameter(self, name: str, param: Optional[Parameter]) -> None:  # type: ignore[return]
         _raise_not_supported(self.register_parameter.__name__)
 
-    def add_module(self, name: str, module: Optional["Module"]) -> None:
+    def add_module(self, name: str, module: Optional["Module"]) -> None:  # type: ignore[return]
```

Comment:
Since you are importing module now, you don't need to escape it in quotes, do you?
```suggestion
    def add_module(self, name: str, module: Optional[Module]) -> None:  # type: ignore[return]
```